packages/framework-provider-azure-infrastructure/src/test-helper/azure-test-helper.ts

```diff
@@ -68,6 +70,11 @@ export class AzureTestHelper {
     return url
   }
 
+  public static async healthURL(resourceGroup: ResourceGroup): Promise<string> {
+    const environment = process.env.BOOSTER_ENV ?? 'azure'
```

Comment:
Does `azure` as default value for the environment have a special meaning?

Reply:
AFAIK it doesn't. It's a copy/paste from `graphqlURL` method. Do you think is safe to remove it?

Reply:
@gonzalogarciajaubert mmm, the `BOOSTER_ENV` var must always be set, but I think it is safer to check it. I would use a common word that doesn't refer to the provider. What do you think about `default` or `booster`?